src/frontend/src/tables/InvenTreeTable.tsx

```diff
@@ -137,9 +182,12 @@ export function InvenTreeTable<T = any>({
         hidden = tableState.hiddenColumns.includes(col.accessor);
       }
 
+      console.log(col.accessor, col.title, fieldNames[col.accessor]);
```

Comment:
Question: Is this left in inventionally?

Reply:
Oops, thanks!